.github/workflows/ci.yml

```diff
@@ -57,7 +57,7 @@ jobs:
             run: |
                 curl -L -O https://tiker.net/ci-support-v0
                 . ci-support-v0
-                ./configure.py --cl-use-shipped-ext
+                export CL_USE_SHIPPED_EXT=on
```

Comment:
Mostly curious @inducer, why aren't the extensions enabled by default? Not very portable?

Reply:
Technically, each system is supposed to have an up-to-date header describing its available CL extensions. That could include extensions that aren't known upstream. Practically, those headers are often out of date and barely maintained. Still, the correct approach is to rely on the system header. Using a shipped header is at best a stopgap.

Reply:
Ah, I see. Thank you for explaining!